lib/livebook_web/live/session_live.ex

```diff
@@ -2272,10 +2272,16 @@ defmodule LivebookWeb.SessionLive do
   end
 
   defp is_secret_on_session?(secret, secrets) do
```

Comment:
Sidenote: should be either `is_` or `?`, in this case `secret_on_session?` :)

Reply:
It could be `secret_toggled?`, WDYT?